### DISCUSSION TITLE
Double entry virtual folder

### DISCUSSION THREAD

**Discussion by kingfisher77** [requarks/wiki#3653] [Error / Bug Report]
I may not understand the folder principle correctly but attached screenshot looks fpr me like a bug.

The path /technology/infrastructure leads to some pages. The path /infrastructure leads to the same pages. What is wrong?

- Wiki.js 2.2.51
- Linux - Centos (core) 8.1.1911 x64
- Node.js 12.16.1
- PostgreSQL 12.2

https://box.kirchhofer.net/s/NAaisRc4Y3MDE9H

**Comment by annahri** [comment]
Same here.

**Comment by kingfisher77** [comment]
Sorry, i accidentally deleted the screenshot. Here is a new one:

https://box.kirchhofer.net/s/LEAKiQRFxy83JiT

**Comment by kingfisher77** [comment]
The "Gost" Pages are gone after install of 2.3.51. Very good!

BTW The new pageTree Navigation is super! We where waiting for that ...
